docs/source/aryn_cloud/get_started.md

```diff
@@ -1,5 +1,5 @@
 ## An Introduction to the Aryn Partitioning Service 
-You can use the Aryn Partitioning Service to easily chunk and extract data from complex PDFs. The Partitioning Service can extract paragraphs, tables and images. It returns detailed information about the components it has just identified in a JSON object.  The following two sections will walk you through two examples where we segment  PDF documents and extract a table and an image from those documents using the python aryn-sdk.
+You can use the Aryn Partitioning Service to easily chunk and extract data from complex documents. The Partitioning Service accepts PDF and Microsoft office formats (.docx, .doc, .pptx, .ppt). It can extract paragraphs, tables and images. It returns detailed information about the components it has just identified in a JSON object.  The following two sections will walk you through two examples where we segment PDF documents and extract a table and an image from those documents using the python aryn-sdk.
```

Comment:
Office isn't used anymore. I'd replace this with "Microsoft Word (.docx and .dox), and Microsoft PowerPoint (.pptx and .ppt)"

Reply:
Done.